src/Arcus.Observability.Correlation/CorrelationInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using GuardNet;
+
+namespace Arcus.Observability.Correlation
+{
+    /// <summary>
+    ///     Represents the correlation ID information on the incoming requests and outgoing responses.
+    /// </summary>
+    public class CorrelationInfo
```

Comment:
This will be the new home for this one @stijnmoreels

Reply:
Then Web API & Messaging can use this and extend it if required.

Reply:
Yes, merged.